tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
+    if command -v invoke-rc.d >/dev/null; then
+        invoke-rc.d kibana stop
+    elif command -v service >/dev/null; then
```

Comment:
They're generated by please-run, - grunt _build:pleaseRun, written to kibana/build/services/sysv/init.d/kibana

```
stop() {
  # Try a few times to kill TERM the program
  if status ; then
    pid=$(cat "$pidfile")
    trace "Killing $name (pid $pid) with SIGTERM"
    kill -TERM $pid
    # Wait for it to exit.
    for i in 1 2 3 4 5 ; do
      trace "Waiting $name (pid $pid) to die..."
      status || break
      sleep 1
    done
    if status ; then
      emit "$name stop failed; still running."
    else
      emit "$name stopped."
    fi
  fi
}
```